Doc/library/pathlib.rst

```diff
@@ -1616,6 +1616,23 @@ Copying, renaming and deleting
       Added return value, return the new :class:`!Path` instance.
 
 
+.. method:: Path.move(target)
```

Comment:
Just me thinking loud, but do you think we should have a boolean to *not* overwrite an existing target (in which case, a `FileExistsError` would be raised)?

Reply:
We could add a `clobber=True` argument, but I think that would be a separate PR.

Reply:
I would have thought of `strict` (strict=False by default, strict=True would raise) or `replace=True`. But we can address this question later. Otherwise, it's just a three liner where users would do `if not os.exists(target)` if they want to avoid replacing files.

Reply:
["Clobber"](https://en.wikipedia.org/wiki/Clobbering) has some currency already, e.g. in `mv --no-clobber`. But yeah, let's discuss later.